benchmark-timed-schedule.yaml

```diff
@@ -0,0 +1,323 @@
+
+  - name: build-survey-runner-benchmark
+    plan:
+    - get: eq-survey-runner-benchmark
+    - get: scheduled-benchmark-trigger
+      trigger: true
+    - put: eq-survey-runner-benchmark-image
+      params:
+        build: eq-survey-runner-benchmark
+        tag_file: eq-survey-runner-benchmark/.git/HEAD
+        tag_as_latest: true
+      get_params:
+        skip_download: true
+    on_failure:
+      put: slack-alert
+      params:
+        channel: '#((slack_channel_name))'
+        attachments:
+          - pretext: Benchmark Image Build Failed
+            color: danger
+            title: Concourse Build $BUILD_ID
+            title_link: $ATC_EXTERNAL_URL/builds/$BUILD_ID
+
+##############################################################################
+# Deploy Infrastructure
+##############################################################################
+
+  - name: deploy-infrastructure
+    serial_groups: [deploy-survey-runner-benchmark]
+    plan:
+    - get: eq-terraform-load-generator
+    - get: scheduled-benchmark-trigger
+      trigger: true
+    - task: Build & Deploy Load Generator
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: google/cloud-sdk
+        params:
+          <<: *common_infrastructure_params
+          IMPORT_EXISTING_PROJECT: ((import_existing_project))
+        inputs:
+          - name: eq-terraform-load-generator
+        run:
+          path: bash
+          args:
+            - -exc
+            - |
+              apt-get install -y unzip
+
+              git clone https://github.com/kamatama41/tfenv.git ~/.tfenv && \
+              ln -s /root/.tfenv/bin/* /usr/local/bin
+
+              cat >$GOOGLE_APPLICATION_CREDENTIALS <<EOL
+              $SERVICE_ACCOUNT_JSON
+              EOL
+
+              gcloud auth activate-service-account --key-file $GOOGLE_APPLICATION_CREDENTIALS
+
+              cd eq-terraform-load-generator
+
+              tfenv install
+
+              ./scripts/deploy_infrastructure.sh
+    on_failure:
+      put: slack-alert
+      params:
+        channel: '#((slack_channel_name))'
+        attachments:
+          - pretext: Load Generator Deploy Failed
+            color: danger
+            title: Concourse Build $BUILD_ID
+            title_link: $ATC_EXTERNAL_URL/builds/$BUILD_ID
+    on_success:
+      put: slack-alert
+      params:
+        channel: '#((slack_channel_name))'
+        attachments:
+          - pretext: Load Generator Deploy Passed
+            color: good
+            title: Concourse Build $BUILD_ID
+            title_link: $ATC_EXTERNAL_URL/builds/$BUILD_ID
+
+##############################################################################
+# Deploy Survey Runner Benchmark
+##############################################################################
+
+  - name: deploy-survey-runner-benchmark
```

Comment:
It's a bit odd to call this deploy, as the action we want to happen is to 'run' the benchmark. I think we should rename this job `run-survey-runner-benchmark`, or even just `run-benchmark`.